### PR TITLE
chore: add SJ translations for What to Expect content

### PR DESCRIPTION
**Pull request by jaredcwhite**
# Pull Request Template

## Issue Overview

This PR addresses #491

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR adds all of the translations mentioned in the issue, which the exception of the confirmation email as that requires a backend change for dev branch (see PR #524)

**NOTE:** there is one translated string still missing, see: https://github.com/housingbayarea/bloom/issues/491#issuecomment-1287591545

## How Can This Be Tested/Reviewed?

View listings and applications that are of a waitlist, lottery, and FCFS type.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by emilyjablonski** [comment]
Why are these translations not added in core?

**Review by emilyjablonski — APPROVED** [review]

File: sites/public/styles/overrides.scss [hunk 1]
```diff
@@ -1,122 +0,0 @@
-.markdown {
```
Comment (emilyjablonski) [inline sites/public/styles/overrides.scss:1]: Curious why this was removed here?

Comment (jaredcwhite) [reply sites/public/styles/overrides.scss:1]: All of those Markdown styles got integrated into Core, which was then pulled back over to HBA: https://github.com/housingbayarea/bloom/blob/dev/ui-components/src/global/markdown.scss

**Comment by jaredcwhite** [comment]
> Why are these translations not added in core? Just a curiosity, they don't seem SJ specific

As I understand it from talking with Kathy, SJ, SMC, and Alameda all have their own workstreams to add the translations in…only English was updated in Core.

**Pull request merged**